tests/cephfs/cephfs_system/mds_admin_daemon_socket_dump.py

```diff
@@ -0,0 +1,118 @@
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1)
+        if not fs_details:
+            fs_util.create_fs(client1, "cephfs")
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ips = fs_util.get_mon_node_ips()
+        fs_util.kernel_mount(
+            [client1],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+        )
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        fs_util.fuse_mount([client1], fuse_mounting_dir_1)
+        cephfs = {
+            "fill_data": 50,
+            "io_tool": "smallfile",
+            "mount": "fuse",
+            "filesystem": "cephfs",
+            "mount_dir": f"{fuse_mounting_dir_1}",
+        }
+        with parallel() as p:
+            p.spawn(fs_io, client=clients[0], fs_config=cephfs, fs_util=fs_util)
+        target_mds = mds_nodes[0]
+        target_mds_name = target_mds.node.hostname
+        dump_list = [
+            "dump cache /",
+            "dump tree /",
+            "dump loads",
+            "dump_blocked_ops",
+            "dump_historic_ops",
+            "dump_historic_ops_by_duration",
+            "dump_mempools",
+            "dump_ops_in_flight",
+            "log dump",
+            "perf dump",
+            "perf histogram dump",
+        ]
+        out1, ec = client1.exec_command(sudo=True, cmd="ceph fs status -f json-pretty")
+        output1 = json.loads(out1)
+        mdsmap = output1["mdsmap"]
+        mds_1 = ""
+        for mds in mdsmap:
+            if target_mds_name in mds["name"]:
+                mds_1 = mds["name"]
+        admin_daemon = (
+            f"cephadm shell ceph --admin-daemon /var/run/ceph/ceph-mds.{mds_1}.asok "
+        )
+        target_mds.exec_command(
+            sudo=True, cmd=f"{admin_daemon} config set debug_mds 10"
+        )
+        target_mds.exec_command(
+            sudo=True, cmd=f"{admin_daemon} config set debug_client 10"
+        )
+        for dump in dump_list:
+            target_mds.exec_command(sudo=True, cmd=f"{admin_daemon} {dump}")
+            out, rc = clients[0].exec_command(sudo=True, cmd="ceph -s -f json")
+            cluster_info = json.loads(out)
+            if cluster_info.get("health").get("status") != "HEALTH_OK":
+                log.error(
+                    f"Cluster helath is in : {cluster_info.get('health').get('status')}"
+                )
+                out, rc = clients[0].exec_command(
+                    sudo=True, cmd="ceph health detail -f json"
+                )
+                error_summary = json.loads(out)
+                log.error(f"{error_summary}")
+                raise CommandFailed("Cluster health is not OK")
```

Comment:
It appears that we may have overlooked the execution of the "ceph fs dump" command, as outlined in the specified Polarion steps. Could you confirm if this omission was intentional?